lib/irb.rb

```diff
@@ -872,12 +821,6 @@
 # IRB.conf[:LC_MESSAGES]
 # :startdoc:
 #
-# :stopdoc:
-# === Single-IRB Mode
```

Comment:
We're deprecating the multi-irb feature soon so let's not introducing the single/multi IRB concept.